Calculator UI

IndieCoderMM opened the pull request:
# Creating a UI for the calculator
The following changes are made in this feature branch:
- Removed extra images
- Created a new component for the calculator UI
- Modified the calculator design in the `index.css`

IndieCoderMM committed:
Remove images

IndieCoderMM committed:
Create calculator ui

IndieCoderMM committed:
Fix linter errors

ridwanediallo commented on src/App.js:
```diff
@@ -1,29 +1,8 @@
-    </div>
-  );
+  return <Calculator />;
 }
```
- [ ] Please replace the app component with a class-based component rather than a functional one. 👌 You will be asked to refactor it into a functional component later in the week. 
Actually, the goal of this is to ensure that you understand how to use both a `class-based component` and a `functional-based component`. It is critical to understand how to use the two because you may encounter code written in either of them in the future.🙂

ridwanediallo commented on src/components/Calculator.js:
```diff
@@ -0,0 +1,53 @@
+    );
+  }
+}
+
```
- [ ] Please make sure the design of your app looks similar when it is compared with the provided example. 🙂Actually, in this case here you have to change the colours by using the ones used in the example.

### Your design 👇️
![Screenshot from 2023-01-09 19-25-23](https://user-images.githubusercontent.com/89228743/211393766-b2b23813-30fb-4101-acd0-26aff09d4eeb.png)

### The example design 👇️
![Screenshot from 2023-01-09 19-41-57](https://user-images.githubusercontent.com/89228743/211393994-2e086177-ad20-4749-ada7-a28b11e6b9ae.png)

ridwanediallo reviewed (changes requested):
# Changes Required ♻️
## Hi @IndieCoderMM, 
![AlmostThereReachGIF](https://user-images.githubusercontent.com/55918904/191301799-36d07166-0e35-4582-a231-7a2d780f7783.gif)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️
- [ ] I would like to suggest that you update the four sections below in your readme file to show the user how to set up and run your project on their local environment, your readme file would be more descriptive as a result. 👇️

![Screenshot from 2023-01-02 17-07-30](https://user-images.githubusercontent.com/89228743/210261706-f19f2417-32bc-41ac-b1d1-e07ca644b686.png)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

IndieCoderMM commented:
Thanks for your detailed feedback. I'll make the required changes.

ridwanediallo commented:
It's a pleasure.

IndieCoderMM committed:
Make required changes

- Change App to a class component
- Change button colors to match the template design
- Update README.md

rustam-tolipov reviewed (approved):
Hi @IndieCoderMM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Developer-512px-10](https://user-images.githubusercontent.com/69011963/151541305-67f4f8e4-c79e-4f9d-a6cf-2c7efc3102c6.gif)

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
No optional suggestions.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
